src/employee_management.c

```diff
@@ -473,6 +473,10 @@ int emp_management(int i,int emp_id)
                 }
                 else if(ch==2)
                 {
+                    int emp_id;
+		    printf("Enter the employee id \n");
+                    scanf("%d",&emp_id);
+                    int k=display_leaves(emp_id);
 
```

Comment:
if Employee does not exist, then dont u think we can show a message that employee id is invalid?

Reply:
Yes we can

Reply:
I raise new request with changed functionality after this